djconnectwise/admin.py

```diff
@@ -170,11 +170,17 @@ class ProjectTypeAdmin(admin.ModelAdmin):
 
 
 @admin.register(models.ProjectPhase)
-class ProjectPhase(admin.ModelAdmin):
+class ProjectPhaseAdmin(admin.ModelAdmin):
     list_display = ('id', 'description', 'project')
     search_fields = ['description']
```

Comment:
It might be a good idea to add `project__name` here too while you are at it, for the same reason as mentioned in my other comment. (This will make more sense if you have seen that other comment first)